src/services/environmentApi.ts

```diff
@@ -0,0 +1,61 @@
+import { AxiosRequestConfig } from 'axios';
+
+import { BuildEnvironmentPage, Environment } from 'pnc-api-types-ts';
+
+import { extendRequestConfig } from 'utils/requestConfigHelper';
+
+import { pncClient } from './pncClient';
+
+export interface IEnvironmentApiData {
+  id: string;
+}
+
+/**
+ * Gets all Environments.
+ *
```

Comment:
Missing TSDoc for `hidden` and `deprecated` parameters and their possible values:
 - `true` - description here
 - `false` - ...
 - `undefined` - ...

But I would consider more self-descriptive values (deprecated context):
 - `WITHOUT_DEPRECATED` (default) - ...
 - `ONLY_DEPRECATED` - ...
 - `ALL` - ...

or

 - `NOT_DEPRECATED_ONLY` (default) - ...
 - `DEPRECATED_ONLY` - ...
 - `ALL` - ...

or

 - `NO` (default) - ...
 - `YES_ONLY` - ...
 - `YES_ALSO` - ...